### DISCUSSION TITLE
[@types/node]: Missing inspector/promises types

### DISCUSSION THREAD

**Discussion by pranshuchittora** [DefinitelyTyped/DefinitelyTyped#70181] [Issues with a @types package] [answered]
Missing types for https://nodejs.org/docs/v20.16.0/api/inspector.html#promises-api
Apparently the same has been mentioned [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/inspector.d.ts#L15) but no following types.

[a comment by an automated account is not shown]

**Comment by Renegade334** [comment, marked as answer]
Added in #70267.
